### ISSUE TITLE
Error CS0234

### ISSUE THREAD

**Issue by E-Krabs** [jgoergen/CamBot#3]
Am I missing something? Visual Studios gives me the following error 'VideoSurveillance does not exist in namespace Emgu.CV' I used NuGet for emgu and zedgraph. I followed instructions but am stuck. Need a little help please. Thank you in advance

**Comment by jgoergen** [comment]
Let me pull this down onto a new computer and try to get it running from scratch and see if I can recreate that, I may need until this weekend but we'll get it figured out!

**Comment by jgoergen** [comment]
I just added a bunch more comments and cleaned up the code a bit, which should include a fix for that problem. Please pull this new code down and give it another try! Don't forget to look at the settings in "VideoSurveilance.cs" incase you need / want to adjust anything.

Please let me know if this works for you.

**Comment by E-Krabs** [comment]
jgoergen, I have been wanting to respond for a while, You are a life saver! I wanted used this for one of my school Sci/English projects. I never knew why they used the word English in a science project but I cant change that. Anyway, I delayed the project for next year as I was exhausted. I was pretty new to C#, so writing my own code was a no go.  So I tried  this repository, it failed on me. But now, not to sure what you did, besides clean the code, it worked! So I plan on using this repo for this years Flox! Just gonna use some older 9g servos and call it a day. Thanks & sorry for really late response.
